README.md

```diff
@@ -33,6 +33,7 @@ Optional:
   - Linux >= 2.6.13
 - [gamin >= 0.0.21](http://www.gnome.org/~veillard/gamin)
 - [systemd >= 204](http://www.freedesktop.org/wiki/Software/systemd)
+- [dnspython or dnspython3](http://www.dnspython.org/)
```

Comment:
I guess no need to specify version specific one since we don't do for the other ones (e.g. pyinotify)